core/foundation/src/StringUtils.cxx

```diff
@@ -37,4 +38,25 @@ std::vector<std::string> Split(std::string_view str, std::string_view delims, bo
    return out;
 }
 
+/// Read and split into white-space separated components a (txt) file
+/// A component string starting with '#' will be ignored
+/// \param[in] str File to read
+/// \param[out] std::vector<std::string> A vector of strings
+std::vector<std::string> ReadFilelist (std::string filelist) {
+    std::vector<std::string> input_file_list;
+    std::fstream infile;
+    infile.open(filelist, std::ios::in);
+    
+    if (infile.is_open()) {
+        infile.seekg(0);
+        std::string line;
+        while( std::getline(infile, line) ) {
```

Comment:
If the file doesn't end on a newline (but the last line contains the last file) then this might skip the last file. Can you check, please?

Reply:
yup, checked with as much variability in the content as possible, the last string is successfully added to the list